experimental/dds/tree2/src/feature-libraries/schemaSummarizer.ts

```diff
@@ -162,7 +170,7 @@ export class SchemaEditor<TRepository extends StoredSchemaRepository>
 	public tryHandleOp(message: ISequencedDocumentMessage): boolean {
 		const op: JsonCompatibleReadOnly = message.contents;
 		if (isJsonObject(op) && op.type === "SchemaOp") {
-			const data = parseSchemaString(op.data as string);
+			const data = this.codec.decode(op.data as string);
```

Comment:
I know its kinda unrelated to your change, but since we are making validation more how we want it eventually, we should probably replace this "as string" with a type narrowing assert so we actually validate it.